package.json

```diff
@@ -41,9 +42,7 @@
   },
   "jsii": {
     "versionFormat": "full",
-    "outdir": "./dist",
     "tsc": {
-      "outDir": "./dist",
```

Comment:
why?

Reply:
otherwise the tests will fail and conflict.

Reply:
The build process does not include packaging as of now and we are relying on the JSII compiler. 

I am bringing in some changes that resolve in another PR I think.